### ISSUE TITLE
Sell Prints total confusing

### ISSUE THREAD

**Issue by hainesmatt** [graphpaperpress/Sell-Media#388]
Sell Prints extension: in the print ordering window, the "add to cart" button is confusing. Normal behavior would be for the user to select the print sizes they want (1 8x10, 3 5x7s etc) of an image, and then click "add to cart" to add all those to the cart. Those values should then clear to 0. The customer could add additional prints of that same image while still viewing the print-order window, or could close the window, or go to the cart window.

Instead, "add to cart" only works when a field is selected that also has a non-zero number in it. The button goes gray and nonfunctional when a different field is selected, that has no value. For example, go here and click "purchase":

http://matthainesphotography.com/galleries/photos/17800024-3/

Type "1" in the 8x10 field. If you don't do anything else, you can click the "add to cart" button. If you tab to the next field, or select another field, you can't add to cart anymore. If you for example type a "3" in the 8-wallets field, you can click "add to cart" again, as long as you're still editing that field. Customers expect to enter in their choices and then be able to click "add to cart" when they're done, no matter what field they're currently editing.

Once you add your selections by clicking the "add to cart" button, it turns into a "checkout" button. The fields are not cleared. The checkout button works, unless you select a field without a value in it, in which case it becomes nonfunctional (just like the add to cart button would). If you enter a value into a field, you can then use the checkout button again...but the item you were trying to edit is not added to the cart!

So to sum up: customers should enter their selections, and then click add to cart without limitation. Those quantities should be cleared, and they are then given the opportunity to add additional prints to their cart if they want. A message should appear, acknowledging that their prints were added to the cart. The "checkout" button is kinda of pointless, because it should always be an "add to cart" button, and clicking on the "cart" link will always allow the customer to check out.
